config/brakeman.ignore

```diff
@@ -1,13 +1,33 @@
 {
   "ignored_warnings": [
+    {
```

Comment:
Are we saying that the word 'role' is what is triggering this?

Reply:
Yeah, that's right 👍